app/src/main/res/layout/fragment_home.xml

```diff
@@ -69,6 +68,8 @@
 
     <android.support.v7.widget.RecyclerView
             android:id="@+id/tileContainer"
+            android:clipToPadding="false"
+            android:paddingTop="10dp"
```

Comment:
If we're adding 10dp of padding here, should we -10dp from the urlbar so the spacing stays the same?

Reply:
o0o0o0oo0o0o0o0o0o0o0o0o0o0o0

Reply:
fwiw, negative values are bad practice in Android (I think it's just because it's confusing) and the IDE should warn you as such.

Reply:
So if we didn't have the extra padding already, how would you do a negative padding? Is there another attribute for it?

Reply:
@mcomella ah, I meant just take 10dp off of the other padding. Not use a negative value! My bad! I should have been more descriptive